From 22cede6771a558eb120191fabf60dfe9af089432 Mon Sep 17 00:00:00 2001
From: Ben Greear <greearb@candelatech.com>
Date: Wed, 30 Dec 2020 15:20:37 -0800
Subject: [PATCH 12/15] intel-oot: Add BusName to debugfs

This gives us an easier way to know the fwcfg
file that may be loaded per device.

Signed-off-by: Ben Greear <greearb@candelatech.com>
---
 drivers/net/wireless/intel-oot/iwlwifi/mvm/debugfs.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/wireless/intel-oot/iwlwifi/mvm/debugfs.c b/drivers/net/wireless/intel-oot/iwlwifi/mvm/debugfs.c
index 5a67ef40fe8a7..e67ce7def70b8 100644
--- a/drivers/net/wireless/intel-oot/iwlwifi/mvm/debugfs.c
+++ b/drivers/net/wireless/intel-oot/iwlwifi/mvm/debugfs.c
@@ -780,6 +780,8 @@ static ssize_t iwl_dbgfs_fw_ver_read(struct file *file, char __user *user_buf,
			 mvm->fwrt.trans->name);
	pos += scnprintf(pos, endpos - pos, "Bus: %s\n",
			 mvm->fwrt.dev->bus->name);
+	pos += scnprintf(pos, endpos - pos, "BusName: %s\n",
+			 dev_name(mvm->fwrt.dev));

	ret = simple_read_from_buffer(user_buf, count, ppos, buff, pos - buff);
	kfree(buff);
--
2.26.3
